app/src/main/java/eu/faircode/netguard/Util.java

```diff
@@ -85,6 +85,7 @@
 import androidx.appcompat.app.AlertDialog;
 import androidx.core.net.ConnectivityManagerCompat;
 
+@SuppressWarnings("WeakerAccess")
```

Comment:
I am no sure what this is about.

Reply:
This suppress the IDE (Android studio) from warning `WeakerAccess`.
For example: a method is decleared with `public` but only used by the class itself. In this situation, that method prefer to be `private`.
In this project, all classes laies under the same package, these methods should be `default`, `package-private` or `private` but `public`.